resources/texts/change_log.txt

```diff
@@ -1,3 +1,65 @@
+[5.10]
+
+* New features and improvements:
+- UltiMaker S8 Support: Added compatibility for the UltiMaker S8 printer.
+- Cheetah Gcode Flavor: Introduced a new Marlin-like Gcode flavor called Cheetah.
+- SpaceMouse support: Interact with your object in 3D thanks to a collaboration with 3Dconnexion
```

Comment:
We already "interact with our objects in 3D" :smile: maybe an other description would be more appropriate ?